### PR TITLE
feat: change SK to token to make primary key printable

### LINKED ISSUE #7: Update Single Table Structure - change Sort Key

### PR DESCRIPTION
**Pull request by makellner**
Fixes #7

### CODE REVIEW

**Comment by hoegertn** [comment]
This should also trigger changes in the Lambda code that uses the new id and changes to the API spec. Or am I missing something?

**Comment by makellner** [comment]
> This should also trigger changes in the Lambda code that uses the new id and changes to the API spec. Or am I missing something?

Changes were added to lambda functions relying on the mail address, additionally it was removed from the API spec.

File: src/lambda/rest.eventmgmtapi.getEventParticipant.ts [hunk 1]
```diff
@@ -5,20 +5,19 @@ import { operations } from '../generated/rest.eventmgmtapi-model.generated';
...
   ctx.logger.info(JSON.stringify(ctx.event));
 
-  const participantId = decodeURIComponent(ctx.event.pathParameters!.participantID!); // id is an e-mail address and thus encoded
   const eventId = ctx.event.pathParameters!.eventID;
   const token = ctx.event.queryStringParameters!.token;
 
-  const participant = await Participant.get({ eventId: eventId, email: participantId });
-  if (!participant || (token !== participant.token && !ctx.auth.isAdmin())) {
+  const participant = await Participant.get({ eventId: eventId, token: token });
+  if (token !== participant?.token && !ctx.auth.isAdmin()) {
```
Comment (hoegertn) [inline src/lambda/rest.eventmgmtapi.getEventParticipant.ts:12]: !participant sollte auch NotFound werfen, dann muss auch das ? unten nicht sein.

Comment (makellner) [reply src/lambda/rest.eventmgmtapi.getEventParticipant.ts]: re-added the condition, also changed that in the other functions, where applicable.

File: src/lambda/rest.eventmgmtapi.updateEventParticipant.ts [hunk 2]
```diff
@@ -6,20 +6,21 @@ export const handler = api.createOpenApiHandlerWithRequestBody<operations['updat
...
-    email: participantId,
+    email: participant.email,
     name: data.name,
     displayName: data.displayName,
     customData: JSON.stringify(data.customData),
   });
-});
\ No newline at end of file
+
+  return new Promise<never>(() => {return JSON.stringify({ status: 'ok' });});
```
Comment (hoegertn) [inline src/lambda/rest.eventmgmtapi.updateEventParticipant.ts:25]: What does this do?

**Review by hoegertn — APPROVED** [review]

**Pull request merged**